dbt/adapters/spark/relation.py

```diff
@@ -27,7 +27,9 @@ class SparkRelation(BaseRelation):
     quote_character: str = "`"
     is_delta: Optional[bool] = None
     is_hudi: Optional[bool] = None
-    information: Optional[str] = None
+    owner: Optional[str] = None
+    stats: Optional[str] = None
+    columns: Dict[str, str] = field(default_factory=lambda: {})
```

Comment:
I think `columns` should be `List[SparkColumn]`, what do you think?

Reply:
@jtcohen6 I think using `List[SparkColumn]` would store a lot of redundant information like `table_database`/`table_schema`/`table_name`... with the benefit of being easier to use, I don't know if it's worth it, but I'll take the advice.